action.yml

```diff
@@ -23,6 +23,10 @@ inputs:
     description: 'A specific repo to sync the secrets to'
     mandatory: False
     default: ''
+  dependabotSecretsSync:
+    description: 'Sync secrets to dependabot (yes/no), default yes'
+    mandatory: False
+    default: 'no'
```

Comment:
this is different from the description above?

Reply:
the description says Yes, the default says No

Reply:
Done